### PR TITLE
update resource

### PR DESCRIPTION
**Pull request by ganisback**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes # 
1. add normal memory resource for llm, add 2gpu resource 
2. support download model with internal domain
3. add default knative domian address

**Type of changes**
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevant result summary. 

- [ ] Test A
- [ ] Test B

- Logs

**Special notes for your reviewer**:

**Checklist**:

- [ ] I have added unit/e2e tests that prove your fix is effective or that this feature works.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have reviewed my own code and ensured that it follows the project's style guidelines.

**Release note**:

```release-note

```

### CODE REVIEW

File: deploy/all_in_one/docker-compose.yml [hunk 1]
```diff
@@ -133,7 +133,7 @@ services:
       STARHUB_SERVER_API_TOKEN: ${HUB_SERVER_API_TOKEN}
       STARHUB_SERVER_S3_ACCESS_KEY_ID: admin
       STARHUB_SERVER_S3_ACCESS_KEY_SECRET: Password_123
-      STARHUB_SERVER_S3_ENDPOINT: ${SERVER_DOMAIN}:9000 # used to generate download links for lfs files
+      STARHUB_SERVER_S3_ENDPOINT: ${SERVER_INTERNAL_DOMAIN:-${SERVER_DOMAIN}}:9000 # used to generate download links for lfs files
```
Comment (wayneliu0019) [inline deploy/all_in_one/docker-compose.yml:136]: 使用这个内部域名生成的下载链接，给到portal后，用户可以下载下来吗？用户客户端得解析出来这个内部域名

Comment (wayneliu0019) [reply deploy/all_in_one/docker-compose.yml:136]: “csghub_server”服务中的`STARHUB_SERVER_S3_ENDPOINT`环境变量会用于组装lfs下载地址，返回给portal，让用户下载，必须得是个公共可访问的url，只能用`SERVER_DOMAIN`吧？  新增加的`SERVER_INTERNAL_DOMAIN`是个内部域名（或者内网ip），用在“mirror-lfs-sync”服务的`STARHUB_SERVER_S3_ENDPOINT`环境变量比较合适，用于多源同步时候往当前csghub的s3写入信息，内部域名效率高。 @pulltheflower  @ganisback

Comment (ganisback) [reply deploy/all_in_one/docker-compose.yml]: Removed this solution, still use public domain to download model

**Review by wayneliu0019 — APPROVED** [review]
LGTM

**Pull request merged**